src/index.html

```diff
@@ -1,13 +1,69 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+
+    <meta name="viewport"
+      content="width=device-width, 
+      user-scalable=no, 
+      initial-scale=1.0, 
+      maximum-scale=1.0, 
+      minimum-scale=1.0"
+    >
+
+    <title>Product cards</title>
+
+    <link rel="stylesheet" href="./style.css">
+
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        class="card__image image" 
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock" 
+      >
+      
+      <h3 class="card__device-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <p class="card__device-code">Product code: 195434</p>
+
+      <div class="container">
```

Comment:
why do we need new block here?
```suggestion
      <div class="card__reviews">
```